airflow.cfg

```diff
@@ -935,7 +935,7 @@ worker_precheck = False
 # See:
 # http://docs.celeryproject.org/en/master/userguide/configuration.html#std:setting-broker_transport_options
 # Example: visibility_timeout = 21600
-# visibility_timeout =
+visibility_timeout =  3600
```

Comment:
this seems fair to me! I'm not aware of any tasks that would need Airflow to wait this long (assuming this is how that works). I also had assessing exactly what is going but I found [this](https://cloud.google.com/composer/docs/composer-2/troubleshooting-dags#troubleshooting_poison_pill) which gives a little more context since the link in the Airflow docs is 404.

Reply:
Parking data ETL tends to run longer than 3600s so I'm not sure if they mean that this timeout should always be greater than our longest ETL? (vs ETA?)

Reply:
extended docs [here](https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/redis.html#id1) but i still don't understand what ETA means in this context 🤷 why are the example settings soooo many hours?

Reply:
Yeah not sure what's going on 
> This causes problems with ETA/countdown/retry tasks where the time to execute exceeds the visibility timeout; in fact if that happens it will be executed again, and again in a loop.

Reply:
ok, [this article](https://engineering.instawork.com/celery-eta-tasks-demystified-424b836e4e94?gi=3ee459f3b272) was slightly more helpful.

my read from this is that if an ETL task runs longer that `visibility_timeout`, there's a risk of it being re-executed by a concurrent worker. since we never want that to happen, we should set this to a high number (21,600s aka 6 hours if that cover's the parking data use case) and also make sure to use the `execution_timeout` [setting](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html#timeouts) to make sure our DAGs timeout before we hit the `visibility_timeout`.

this will definitely be a setting to keep an eye on once we launch. and if we are confounded by the CeleryExecutor we can always revert to the simple `SequentialExecutor`.